osu.Game.Tests/Visual/Collections/TestSceneManageCollectionsDialog.cs

```diff
@@ -141,7 +141,8 @@ public void TestCollectionNameCollisions()
         {
             AddStep("add dropdown", () =>
             {
-                Add(new CollectionFilterDropdown
+                Add(
+                    new CollectionFilterDropdown
```

Comment:
Dislike this change, old code was preferable.

Reply:
This is tricky to satisfy both analyzers.

Reply:
Something like this is idiomatic in the codebase and doesn't seem to trigger the .net analyser:

```diff
diff --git a/osu.Game.Tests/Visual/Collections/TestSceneManageCollectionsDialog.cs b/osu.Game.Tests/Visual/Collections/TestSceneManageCollectionsDialog.cs
index d5983ac827..888002eb36 100644
--- a/osu.Game.Tests/Visual/Collections/TestSceneManageCollectionsDialog.cs
+++ b/osu.Game.Tests/Visual/Collections/TestSceneManageCollectionsDialog.cs
@@ -141,15 +141,13 @@ public void TestCollectionNameCollisions()
         {
             AddStep("add dropdown", () =>
             {
-                Add(
-                    new CollectionFilterDropdown
-                    {
-                        Anchor = Anchor.TopRight,
-                        Origin = Anchor.TopRight,
-                        RelativeSizeAxes = Axes.X,
-                        Width = 0.4f,
-                    }
-                );
+                Add(new CollectionFilterDropdown
+                {
+                    Anchor = Anchor.TopRight,
+                    Origin = Anchor.TopRight,
+                    RelativeSizeAxes = Axes.X,
+                    Width = 0.4f,
+                });
             });
             AddStep("add two collections with same name", () => manager.Collections.AddRange(new[]
             {
```

Reply:
I remember that resharper requires 4 chars of indentation of the braces here, while VS requires no indentation.

Reply:
Yes, and I'm fixing that conflict by merging closing curly and round brackets into a single line. Which makes the analysers agree again.